test/NuGet.Core.Tests/NuGet.Protocol.Tests/Utility/MetadataReferenceCacheTestUtility.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Please revert this. It's an unnecessary change.

Reply:
Done.